website/src/components/ExampleSlideshow/images/right-hand-drawn-arrow.svg

```diff
@@ -0,0 +1,14 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 75 180">
+  <defs>
+    <style>
+      @font-face {
+        font-family: "Virgil";
+        src: url("https://excalidraw.com/FG_Virgil.ttf");
+      }
+      @font-face {
+        font-family: "Cascadia";
+        src: url("https://excalidraw.com/Cascadia.ttf");
+      }
```

Comment:
Hmm... why are there font includes inside the arrow svg?

Reply:
great catch! -- i used excalidraw to create them and I guess it includes these by default.

will def remove them.